Build failure is
```
2.1.3 -Wl,-z,max-page-size=4096 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro -Wl,-z,pack-relative-relocs " PYO3_CROSS_LIB_DIR="/builder/staging_dir/target-i386_pentium-mmx_musl/usr/lib/python3.14" SETUPTOOLS_RUST_CARGO_PROFILE="release"  LINK_SYSTEM_LIBGPIOD=1  /builder/staging_dir/hostpkg/bin/python3.14  -m build --no-isolation --outdir "/builder/build_dir/target-i386_pentium-mmx_musl/libgpiod-2.1.3/bindings/python"/openwrt-build --wheel   "/builder/build_dir/target-i386_pentium-mmx_musl/libgpiod-2.1.3/bindings/python" 
2026-04-27T14:20:14.9724038Z * Getting build dependencies for wheel...
2026-04-27T14:20:15.2175629Z 
2026-04-27T14:20:15.2176203Z Traceback (most recent call last):
2026-04-27T14:20:15.2176863Z   File "/builder/staging_dir/hostpkg/lib/python3.14/site-packages/pyproject_hooks/_impl.py", line 402, in _call_hook
2026-04-27T14:20:15.2177437Z     raise BackendUnavailable(
2026-04-27T14:20:15.2177661Z     ...<4 lines>...
2026-04-27T14:20:15.2177835Z     )
2026-04-27T14:20:15.2178147Z pyproject_hooks._impl.BackendUnavailable: Cannot import 'setuptools.build_meta'
2026-04-27T14:20:15.2178460Z 
2026-04-27T14:20:15.2178939Z ERROR Backend 'setuptools.build_meta:__legacy__' is not available.
2026-04-27T14:20:15.2351045Z make[2]: *** [Makefile:170: /builder/build_dir/target-i386_pentium-mmx_musl/libgpiod-2.1.3/.built] Error 1
2026-04-27T14:20:15.2351674Z make[2]: Leaving directory '/feed/libs/libgpiod'
2026-04-27T14:20:15.2354215Z time: package/feeds/packages_ci/libgpiod/compile#55.80#9.30#23.15
2026-04-27T14:20:15.2363399Z     ERROR: package/feeds/packages_ci/libgpiod failed to build.
2026-04-27T14:20:15.2370690Z make[1]: *** [package/Makefile:188: package/feeds/packages_ci/libgpiod/compile] Error 1
2026-04-27T14:20:15.2371385Z make[1]: Leaving directory '/builder'
2026-04-27T14:20:15.2376162Z make: *** [/builder/include/toplevel.mk:226: package/libgpiod/compile] Error 2
2026-04-27T14:20:15.2380511Z ##[group]  make package/index
```


This may need  a `PKG_BUILD_DEPENDS:=python-setuptools/host`  addition.

--- a/libs/libgpiod/Makefile
+++ b/libs/libgpiod/Makefile
@@ -23,6 +23,8 @@ PKG_INSTALL:=1
 PKG_BUILD_PARALLEL:=1
 PYTHON3_PKG_BUILD:=0
 
+PKG_BUILD_DEPENDS:=PACKAGE_python3-gpiod:python-setuptools/host
+
 include $(INCLUDE_DIR)/package.mk
 include ../../lang/python/python3-package.mk
 

--- a/libs/libgpiod/Makefile
+++ b/libs/libgpiod/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libgpiod
 PKG_VERSION:=2.1.3
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=@KERNEL/software/libs/libgpiod/
@@ -90,7 +90,7 @@ define Package/python3-gpiod
   SUBMENU:=Python
   TITLE:=Python bindings for libgpiod
   URL:=https://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git
-  DEPENDS:=+python3-light +libgpiod
+  DEPENDS:=+PACKAGE_python3-gpiod:python3-light +libgpiod
 endef
 
 define Package/python3-gpiod/description

